src/profile/components/basicData/__tests__/BasicData.test.tsx

```diff
@@ -26,36 +35,157 @@ describe('<BasicData /> ', () => {
       </RenderChildrenWhenDataIsComplete>
     );
   };
+  const t = i18n.getFixedT('fi');
+  const editButtonSelector: ElementSelector = { id: 'basic-data-edit-button' };
+  const cancelButtonSelector: ElementSelector = {
```

Comment:
cancelButtonSelector seems to be only used in the last test. Would it be increase readability to move it inside the last test, as it seems not to be needed elsewhere yet?

Reply:
I thought it was cleaner to group selectors in one place. I'll change this.

Reply:
Done.